lib/reporters/tap.js

```diff
@@ -62,17 +62,23 @@ class TapReporter {
 
 		const title = stripAnsi(test.title);
 
+		const appendLogs = () => {
+			if (test.logs) {
+				test.logs.forEach(log => {
+					output.push(`  ${log}`);
```

Comment:
If `log` contains linebreaks, should the indentation apply on those new lines? If so, this should use https://www.npmjs.com/package/indent-string (like elsewhere in the reporters). May not be a bad idea to use that anyway.